pkg/dns/gcp/gcp.go

```diff
@@ -0,0 +1,71 @@
+package gcp
+
+import (
+	"context"
+
+	iov1 "github.com/openshift/cluster-ingress-operator/pkg/api/v1"
+	"github.com/openshift/cluster-ingress-operator/pkg/dns"
+	logf "github.com/openshift/cluster-ingress-operator/pkg/log"
+
+	configv1 "github.com/openshift/api/config/v1"
+
+	gdnsv1 "google.golang.org/api/dns/v1"
+	"google.golang.org/api/option"
+)
+
+var (
+	_   dns.Provider = &Provider{}
+	log              = logf.Logger.WithName("dns")
```

Comment:
Unused.  Presumably we should have some logging though.

Reply:
Removed